src/components/Menu.vue

```diff
@@ -255,9 +255,12 @@ export default {
     },
     joinSession() {
       if (this.session.sessionId) return this.leaveSession();
-      const sessionId = prompt(
+      let sessionId = prompt(
         "Enter the channel number / name of the session you want to join"
       );
+      if (sessionId.match(/^https?:\/\//i)) {
+        sessionId = sessionId.split("#")[1];
```

Comment:
Sorry for nitpicking again, but if someone pasted a link that doesn't contain a `#`, this would throw an error. So it's safer to just use this:
```suggestion
        sessionId = sessionId.split("#").pop();
```

Reply:
good shout, I have updated this.